generators/binarygen/analysis/analysis.go

```diff
@@ -0,0 +1,511 @@
+// Package analysis uses go/types and go/packages to extract
+// information about the structures to convert to binary form.
+package analysis
+
+import (
+	"fmt"
+	"go/ast"
+	"go/constant"
+	"go/types"
+	"path/filepath"
+	"reflect"
+	"strings"
+
+	"golang.org/x/tools/go/packages"
+)
+
+// Analyser provides information about types,
+// shared by the parser and writer code generator
+type Analyser struct {
+	// Source is the path of the origin go source file.
+	Source string
+
+	sourceAbsPath string
+
+	pkg *packages.Package
+
+	// Tables contains the resolved struct definitions, coming from [Sources]
```

Comment:
What is `[Sources]` in this context? The closest matching field is `Source`. Is this perhaps leftover from a refactor?

Reply:
It was a leftover indeed, thank you. I've fixed the comment.